import_export/results.py

```diff
@@ -1,9 +1,13 @@
+import logging
 from collections import OrderedDict
 
 from django.core.exceptions import NON_FIELD_ERRORS
 from django.utils.encoding import force_str
+from django.utils.translation import gettext_lazy as _
```

Comment:
Is there a reason we import this as ' _ '? Is there is some reason its just underscore, and not a more descriptive name that I don't understand?

Reply:
It's used as a shortcut for [gettext()](https://docs.djangoproject.com/en/5.0/topics/i18n/translation/#standard-translation).  I imagine that writing the full function name may end up cluttering the code unnecessarily, so you are allowed to use a shortcut by convention.